.gitignore

```diff
@@ -3,3 +3,5 @@ node_modules/
 dist/
 npm-debug.log
 yarn-error.log
+coverage/
```

Comment:
Apparently they (webpack boilerplate) are following the Tooling conventions:
• Istanbul Coverage (used by them), places it under `./coverage`:
https://github.com/gotwarlost/istanbul#getting-started

Should I change it?